backend/src/main/kotlin/ch/yvu/teststore/Application.kt

```diff
@@ -4,13 +4,25 @@ import org.springframework.boot.SpringApplication
 import org.springframework.boot.autoconfigure.EnableAutoConfiguration
 import org.springframework.boot.autoconfigure.SpringBootApplication
 import org.springframework.boot.autoconfigure.jdbc.DataSourceAutoConfiguration
+import org.springframework.context.annotation.Bean
 import org.springframework.scheduling.annotation.EnableAsync
+import org.springframework.web.servlet.config.annotation.CorsRegistry
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurer
 
 @SpringBootApplication
 @EnableAutoConfiguration(exclude = arrayOf(DataSourceAutoConfiguration::class))
 @EnableAsync
-open class Application
+open class Application {
+    @Bean
+    open fun corsConfigurer(): WebMvcConfigurer {
+        return object : WebMvcConfigurer {
+            override fun addCorsMappings(registry: CorsRegistry) {
```

Comment:
I would only activate CORS once we there is proper authentication (e.g. auth tokens) implemented

Reply:
Commit removed.